robyn/dev_event_handler.py

```diff
@@ -11,7 +12,8 @@ def __init__(self, file_name) -> None:
     def start_server_first_time(self) -> None:
         if self.processes:
             raise Exception("Something wrong with the server")
-        self.processes.append(subprocess.Popen(["python3", self.file_name], start_new_session=False))
+        command = "python3" if not sys.platform.startswith("win32") else "python"
```

Comment:
instead of this we can create a private computed python property that gives us a python alias depending on the platform. What do you think?

Reply:
Correct. That would be much better.